packages/x-components/src/views/home/Home.vue

```diff
@@ -349,7 +351,10 @@
             </LocationProvider>
 
             <!-- Partials -->
-            <PartialResultsList :animation="resultsAnimation">
+            <PartialResultsList
+              v-if="!$x.isNoResultsWithFilters && ($x.totalResults <= 4 || $x.isNoResults)"
```

Comment:
```suggestion
              v-if="!$x.fromNoResultsWithFilters && ($x.totalResults <= 4 || $x.isNoResults)"
```

Reply:
Note that the alias API exposes `noResults` instead of `isNoResults`.

```suggestion
              v-if="!$x.isNoResultsWithFilters && ($x. fromNoResultsWithFilters <= 4 || $x.noResults)"
```